mParticle-Kochava/MPKitKochava.m

```diff
@@ -1,18 +1,12 @@
 #import "MPKitKochava.h"
 #import "MPKochavaSpatialCoordinate.h"
-#import "mParticle.h"
-#import "MPKitRegister.h"
-#import "MPEnums.h"
+//#import "MPKitRegister.h"
+//#import "MPEnums.h"
```

Comment:
```suggestion
```
We should get rid of these comments if we're not using the headers.

Reply:
thanks for catching this